.flowconfig

```diff
@@ -8,3 +8,6 @@
 <PROJECT_ROOT>/node_modules/jss
 <PROJECT_ROOT>/node_modules/findup
 <PROJECT_ROOT>/node_modules/editions
+
+[options]
```

Comment:
Added to avoid having CircleCI issues like this one: https://circleci.com/gh/Khan/wonder-blocks/2076?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link